Example/Tests/Tests.swift

```diff
@@ -83,28 +82,6 @@ class Tests: XCTestCase {
         XCTAssertEqual(Utils.scrubScopes(scopes), ["profile", "email", "openid"])
     }
 
-    func testPasswordFailureFlow() {
```

Comment:
Why was this test removed?

Reply:
In order not to fix it while we decided to remove this functionality at all. https://github.com/okta/okta-sdk-appauth-ios/pull/80#discussion_r239690514

Reply:
Since this PR doesn't remove the password flow feature, we shouldn't be removing test cases that assert the feature works as intended.

There is another PR that fixes this test (see [this line](https://github.com/okta/okta-sdk-appauth-ios/pull/87/files#diff-51059c73416c73e57fe0ce5280f66e76R97)). Can we please bring back this test case until #82 is implemented?